daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala

```diff
@@ -49,6 +49,10 @@ object Ref {
   type CommandId = LedgerString
   val CommandId: LedgerString.type = LedgerString
 
+  /** Identifiers used to correlate an admin submission with results. */
```

Comment:
Possibly an ignorant question but why an "admin submission|?

Reply:
Why only for admin ones though?

Reply:
I think the submission ID was historically only used for the admin interface. However, on closer inspection, we use it everywhere. So I should probably change that.

Reply:
Indeed, now we have submission ID on all `WriteService` calls (for transactions it used to only have command ID in v1).

Reply:
I've changed the comment.